packages/features/ee/teams/lib/payments.ts

```diff
@@ -117,10 +117,8 @@ export const updateQuantitySubscriptionFromStripe = async (teamId: number) => {
       return;
     }
 
-    const newQuantity = membershipCount - subscriptionQuantity;
-
     await stripe.subscriptions.update(subscriptionId, {
-      items: [{ quantity: membershipCount + newQuantity, id: subscriptionItemId }],
+      items: [{ quantity: membershipCount, id: subscriptionItemId }],
```

Comment:
membershipCount was 3 but subscriptionQty was 2 so this will result in 4 instead of 3.
```suggestion
     const newQuantity = membershipCount - subscriptionQuantity; // 3 - 2 = 1
     ....
     items: [{ quantity: membershipCount + newQuantity, id: subscriptionItemId }], // 3 + 1 = 4 Which is wrong
```